React/wbooks/src/services/bookService.js

```diff
@@ -0,0 +1,27 @@
+import api from '../app/config/api';
+
+import * as localStorage from './localstorageService.js';
+
+export const retrieveUserFromSession = () => {
+  return localStorage.retrieveUserFromLocalStorage();
+}
+
+export const getAllBooks = async () => {
+  return await api.get('/books', {
+      'headers': { 'Authorization': retrieveUserFromSession() }
+  }).then(response => {
+    return response;
+  }).catch(error => {
+    throw error;
+  });
+}
```

Comment:
you can simplify it in the following way...

```javascript
export const getAllBooks = () => api.get('/books');
```

then, in the screen...

```javascript
fetchBooks = async () => {
  const response = await bookServices.getAllBooks();
  if (response.ok) {
    // Do something if request is sucessfull
  } 
  else {
    // Do if error
  }
}
```

Reply:
Other comment, set a header in each endpoint is wrong. You must set the header when login is satisfactory.